include/envoy/tracing/trace_driver.h

```diff
@@ -6,6 +6,7 @@
 
 #include "envoy/common/pure.h"
 #include "envoy/stream_info/stream_info.h"
+#include "envoy/tracing/trace_context.h"
```

Comment:
Is this really needed?

Reply:
Yes, for now this is still not needed and I will remove it.